pb/Makefile

```diff
@@ -0,0 +1,11 @@
+PB = $(wildcard *.proto)
+GO = $(PB:.proto=.pb.go)
+
+all: $(GO)
+
+%.pb.go: %.proto
+		protoc --proto_path=$(GOPATH)/src:. --gogofast_out=. $<
+
+clean:
+		rm -f *.pb.go
+		rm -f *.go
```

Comment:
```suggestion
		rm -f *.go
.PHONY: clean
```

Reply:
I removed `rm -f *.go` because we now have a custom go file in that dir. But I added the phony target.